### PR TITLE
src/components/coordinator: add component TaskListCoordinator to display task timeline

### PR DESCRIPTION
**Pull request by maceksimon**
Add `TaskListCoordinator` component that shows the timeline of tasks that coordinator needs to complete.
Allows to show/hide past tasks and displays a month label for first task of a month.

* Add logic for computing labels and past/future items in an inline composable
* Add translations
* Add tests

### CODE REVIEW

File: src/components/coordinator/TaskListCoordinator.vue [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+          month = taskMonth;
+        });
+
+        return tasks;
+      });
+
+      /**
+       * Filter future tasks
+       */
+      const taskListFuture = computed(() => {
```
Comment (tmszi) [inline src/components/coordinator/TaskListCoordinator.vue:88]: Returned value type?

File: src/components/coordinator/TaskListCoordinator.vue [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        return taskList.value.filter((task: TaskCoordinator) => {
+          return new Date(task.date) > new Date();
+        });
+      });
+
+      /**
+       * Filter past tasks
+       */
+      const taskListPast = computed(() => {
+        return taskList.value.filter((task: TaskCoordinator) => {
```
Comment (tmszi) [inline src/components/coordinator/TaskListCoordinator.vue:98]: Returned value type?

**Review by tmszi — APPROVED** [review]

File: src/components/coordinator/TaskListCoordinator.vue (hunk 1 above)
Comment (maceksimon) [reply src/components/coordinator/TaskListCoordinator.vue]: Added.

File: src/components/coordinator/TaskListCoordinator.vue (hunk 2 above)
Comment (maceksimon) [reply src/components/coordinator/TaskListCoordinator.vue:98]: Added.

**Pull request merged**